### DISCUSSION TITLE
jbrowse properly installed but embedding website loads nothing

### DISCUSSION THREAD

**Discussion by samuelmiver** [GMOD/jbrowse-components#3195] [Q&A] [answered]
Hi! Some weeks ago I successfully installed and run the tutorial to embed jbrowser using the [tutorial](https://jbrowse.org/jb2/docs/tutorials/embed_linear_genome_view/01_introduction/). 

However, I tried to run it again by `npx serve` in the directory with the index, assembly and tracks file and it shows nothing, only the html header. I have tried repeating step by step and same result, no browser is displayed. 

**additional info:**
jbrowse is installed in `/usr/local/bin/jbrowse` (ubuntu via WSL installation)
When running `jbrowse --version` I get `@jbrowse/cli/2.1.2 wsl-x64 node-v18.8.0`
Finally, running `jbrowse create jbrowse2` and serving the directory gives me the volvox example without any problem

The directory is composed by the three files present in [reference](https://jbrowse.org/jb2/docs/tutorials/embed_linear_genome_view/09_conclusion/) section.

Any help will be much appreciated!

**Comment by garrettjstevens** [comment, marked as answer]
When you load the page, are there any messages in the [browser console](https://developer.chrome.com/docs/devtools/open/#console)?

**Reply by samuelmiver** [reply]
Yes, the two following errors are reported:

```
Failed to load resource: the server responded with a status of 404 ()
unpkg.com/@jbrowse/react-linear-genome-view@2.1.3/dist/react-linear-genome-view.umd.development.js:1 


Uncaught ReferenceError: JBrowseReactLinearGenomeView is not defined at (index):30:9
```

Thanks in advance!

**Reply by garrettjstevens** [reply]
It looks like our latest release is missing some necessary files. For now, you can change the `script` tag in `index.html` to specify a previous version, like this:

```html
<script
  src="//unpkg.com/@jbrowse/react-linear-genome-view@2.1.2/dist/react-linear-genome-view.umd.development.js"
  crossorigin
></script>
```

**Reply by cmdcolin** [reply]
the code in the tutorial (referring to latest) should now work with 2.1.4 (2.1.3 was indeed a botched release for the umd bundles!) thanks for reporting @samuelmiver !

**Reply by samuelmiver** [reply]
Moving to the previous version fixed the problem, thanks @garrettjstevens!

**Reply by samuelmiver** [reply]
> the code in the tutorial (referring to latest) should now work with 2.1.4 (2.1.3 was indeed a botched release for the umd bundles!) thanks for reporting @samuelmiver !

Tested as well, fully functional now. Thanks to you @cmdcolin ;) 

Best!

**Reply by garrettjstevens** [reply]
@agustin-bilat you seem to be using JBrowse Web, where the original issue in this discussion was with the embedded version of JBrowse, so the solution likely won't be the same. You might want to check if `serve` is using port 3000, since it may end up using a different port. Also you could check the browser's developer tools console to look for any errors. If neither of those help, it would probably be best to [create a new discussion](https://github.com/GMOD/jbrowse-components/discussions/new/choose) to track this.
